### DISCUSSION TITLE
fit_model returning NaN

### DISCUSSION THREAD

**Discussion by kharelg1** [georgebv/pyextremes#47] [General]
I am getting "-inf" for the return value and NaN for the confidence intervals after running the fit_model() - see attached file (issue1). 
![issue-1](https://user-images.githubusercontent.com/126341241/221386015-fa30c2d6-e584-469d-b0b9-3ccfe36d9d88.PNG)
I used the following script for the attached data:
    model = EVA(df)
    model.from_extremes(df, method='BM',extremes_type='high', block_size='365.2425D')
    model.set_extremes(df)
    model.plot_extremes(figsize=(8,5), ax=None, show_clusters=False)
    fitting = model.fit_model(model='MLE', distribution='genextreme')
    print (fitting)
    r_period = [1,25,100,500,1000]
    summary = model.get_summary(r_period, return_period_size='365.2425D', alpha=0.95, n_samples=10000)
    print (summary)
[test_annual_max_series.txt](https://github.com/georgebv/pyextremes/files/10832622/test_annual_max_series.txt)

I would appreciate it if you could help in finding anything wrong with the script and the output.

**Comment by georgebv** [comment]
This is expected when using block size of 1 year and looking for 1-year return value. Effectively, you are asking a question of "what is the value which has a 100% probability of being exceeded". An answer to that is naturally negative infinity. I recommend reading about GEVD to understand why this is happening.
